spec/classes/init_spec.rb

```diff
@@ -47,6 +100,31 @@
     end
   end
 
+  describe "when repotype is set to apt" do
+    let(:params) { { :repotype => 'apt', :repos => repos_apt_hash, :apt_setting => apt_setting_hash } }
```

Comment:
For better readability, please don't define more than 1 or 2 parameters in one line.